cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -383,28 +391,33 @@ They use in-memory key storage and the Tink crypto library.
 #### Kotlin (Android)
 
 ```kotlin
-import dev.trustbloc.wallet.sdk.api.*
-import dev.trustbloc.wallet.sdk.did.*
+import dev.trustbloc.wallet.sdk.localkms.Localkms
+import dev.trustbloc.wallet.sdk.localkms.SignerCreator
+import dev.trustbloc.wallet.sdk.did.Resolver
+import dev.trustbloc.wallet.sdk.did.Creator
+import dev.trustbloc.wallet.sdk.ld.DocLoader
 import dev.trustbloc.wallet.sdk.localkms
 import dev.trustbloc.wallet.sdk.openid4vp
 import dev.trustbloc.wallet.sdk.ld
 import dev.trustbloc.wallet.sdk.credential
 
 // Setup
-val kms = localkms.KMS(null) // The null store argument causes it to use in-memory storage. Will use the Tink crypto library.
-val didResolver = did.Resolver()
-val docLoader = ld.DocLoader()
-val didCreator = did.NewCreatorWithKeyWriter(kms)
-val didDocResolution = didCreator.create("key", api.CreateDIDOpts()) // Create a did:key doc
+val kms = Localkms.newKMS(null)// The null store argument causes it to use in-memory storage. Will use the Tink crypto library.
+val signerCreator = Localkms.createSignerCreator(kms) // Will use the Tink crypto library
+val didResolver = Resolver("")
+val didCreator = Creator(kms as KeyWriter)
+val documentLoader = DocLoader()
+val didDocResolution = didCreator.create("key", CreateDIDOpts()) // Create a did:key doc
 
 // Going through the flow
 val interaction = openid4vp.Interaction("YourAuthRequestURIHere", kms, kms.getCrypto(), didResolver, docLoader)
 val query = interaction.getQuery()
 val inquirer = credential.Inquirer(docLoader)
 val issuedCredentials = api.VerifiableCredentialsArray() // Would need some actual credentials for this to actually work
 val verifiablePres = inquirer.Query(query, credential.CredentialsOpt(issuedCredentials))
+val matchedCreds = verifiablePresentation.credentials() // this is to show on comfimation dialog.
```

Comment:
Typo in `comfimation` - should say `confirmation`.

Also, is this message specific to our reference app? If so, I don't think it belongs here as this is general-purpose documentation. Perhaps you can word it in a more general way.

Reply:
I would recommend changing the wording to something like "// This should be shown to the user with a confirmation dialog so they can confirm that they want to share this data before calling presentCredential` to make the intent more clear to the user of the SDK.